storage/s3-eks-public/vars.tf

```diff
@@ -31,3 +31,19 @@ variable "additional_tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "acl" {
+  description = "The ACL is the S3 bucket security settings"
+  type        = string
+  default     = "public-read"
+
+  validation {
+    condition     = (var.acl == "public-read" || var.acl == "private")
+    error_message = "Bucket ACL must be either public-read private. Defaults to public-read. Should be private for sandboxes."
+  }
+}
+
+variable "eks_is_sandbox" {
```

Comment:
We should probably use this in the `sandbox-template`.

Reply:
Yes, we must use this in the template, because it needs to be defined at the highest level, in account.tfvars :)

Reply:
Done